its/tests/src/test/java/its/SonarQubeEnterpriseEditionTests.java

```diff
@@ -94,7 +108,9 @@ class SonarQubeEnterpriseEditionTests extends AbstractConnectedTests {
 
   private static SonarLintRpcServer backend;
 
-  private static BackendJsonRpcLauncher serverLauncher;
+  private static final List<String> didSynchronizeConfigurationScopes = new CopyOnWriteArrayList<>();
+
+  private static final List<LogParams> clientLogs = new CopyOnWriteArrayList<>();
```

Comment:
Note that I think using a ConcurrentLinkedDeque is better than a CopyOnWriteArrayList for something having as many writes as logs.